### PR TITLE
SPLAT-1381: Remove Alibaba

### PR DESCRIPTION
**Pull request by jcpowermac**
**- What I did**

Alibaba is no longer available for an IPI installation, removed all references to Alibaba (except vendoring).

**- How to verify it**

**- Description for the changelog**

Alibaba is no longer available for an IPI installation, removed all references to Alibaba (except vendoring).
Updated unit tests replacing AlibabaPlatform with Example.

### CODE REVIEW

**Comment by jcpowermac** [comment]
cc: @faermanj just making sure you don't think this will break an AI install of Alibaba

**Comment by jcpowermac** [comment]
tested locally no issues, retrying

/test unit

File: templates/common/alibabacloud/OWNERS [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
-# This file just uses aliases defined in OWNERS_ALIASES.
-
-approvers:
```
Comment (yuqi-zhang) [inline templates/common/alibabacloud/OWNERS:4]: I think these are still being defined in `OWNERS_ALIASES` ?

**Review by yuqi-zhang — APPROVED** [review]
Seems fine offhand. Is there a corresponding PR to remove this from the API?

Also seems to be 1 reference still (comment inline)

**Comment by jcpowermac** [comment]
> Is there a corresponding PR to remove this from the API?

Not yet, that will be one of the last things we do

**Pull request merged**